regiontag_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package samples
+
+import (
+	"bufio"
+	"bytes"
+	"log"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestRegionTags(t *testing.T) {
+	type RegionTag struct {
+		tag  string
+		path string
+	}
+	errs := []RegionTag{}
+
+	cmd := exec.Command("go", "list", "./...")
+	out, err := cmd.Output()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	scanner := bufio.NewScanner(strings.NewReader(string(out)))
```

Comment:
Use `bytes.NewReader` or hook directly into `cmd.StdoutPipe()`?

Something like this (untested):
```go
goDoc := exec.Command("go", "list", "./...")
out, err := goDoc.StdoutPipe()
if err != nil {
  t.Fatalf(err)
}
scanner := bufio.NewScanner(out)
```

Reply:
Done!